src/frontends/pytorch/src/op/convolution.cpp

```diff
@@ -11,42 +11,85 @@ namespace frontend {
 namespace pytorch {
 namespace op {
 
+Output<ov::Node> reshape_bias(NodeContext& context, Output<ov::Node> bias, Output<ngraph::Node> conv) {
+    auto conv_shape = context.mark_node(std::make_shared<opset8::ShapeOf>(conv));
+    auto conv_rank = context.mark_node(std::make_shared<opset8::ShapeOf>(conv_shape));
+    auto one_const = context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {1}));
+    auto two_const = context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {2}));
+    auto tail_shape_rank = context.mark_node(std::make_shared<opset8::Subtract>(conv_rank, two_const));
+    auto tail_shape = context.mark_node(std::make_shared<opset8::Broadcast>(one_const, tail_shape_rank));
+    auto channels_dim = context.mark_node(std::make_shared<opset8::ShapeOf>(bias));
+    auto new_shape =
+        context.mark_node(std::make_shared<opset8::Concat>(OutputVector{one_const, channels_dim, tail_shape}, 0));
+
+    return context.mark_node(std::make_shared<opset8::Reshape>(bias, new_shape, false));
+}
+
 OutputVector translate_convolution(NodeContext& context) {
     // Shchema: aten::_convolution(Tensor input, Tensor weight, Tensor? bias, int[] stride, int[] padding, int[]
     // dilation, bool transposed, int[] output_padding, int groups, bool benchmark, bool deterministic, bool
     // cudnn_enabled, bool allow_tf32) -> Tensor
-    bool transposed = context.const_input<bool>(6);
-    // TODO: Handle this temporary limitation
-    OV_FRONTEND_REQUIRE(!transposed);
 
     auto strides = context.const_input<Strides>(3);
-    auto pads_begin = context.const_input<CoordinateDiff>(4);  // FIXME: The same 4 is used twice
-    auto pads_end = context.const_input<CoordinateDiff>(4);    // FIXME: The same 4 is used twice
+    auto pads = context.const_input<CoordinateDiff>(4);
```

Comment:
String provided pads aren't supported here, can we at least have TODO comment about that here?

Reply:
it is not TO DO, function signature does not allow to use string (for this case convolution_mode used)